intake_esm/collection_defs/cesm.yaml

```diff
@@ -1,28 +1,23 @@
 collections:
   cesm:
     collection_columns:
-    - resource
-    - resource_type
-    - direct_access
     - experiment
     - case
     - component
     - stream
     - variable
     - date_range
     - member_id
-    - file_fullpath
-    - file_basename
+    - path
     - ctrl_branch_year
     - year_offset
     - sequence_order
-    - grid
     order_by_columns:
       - sequence_order
```

Comment:
I am going to revisit the CESM collection once I've got CMIP6 fully working. I will eventually remove this column. I kept it because I didn't want to break CESMCollection for the time being.